tools/cli/flags.go

```diff
@@ -441,8 +443,8 @@ func getFlagsForDescribe() []cli.Flag {
 func getFlagsForDescribeID() []cli.Flag {
 	return []cli.Flag{
 		cli.BoolFlag{
-			Name:  FlagPrintRawTimeWithAlias,
-			Usage: "Print raw time stamp",
+			Name:  FlagPrintRawWithAlias,
+			Usage: "Print raw time stamp and properties as they are",
```

Comment:
and **other** properties

Or maybe just say "print raw properties as they are stored".

Reply:
changed